content/sensu-go/5.18/guides/clustering.md

```diff
@@ -258,11 +258,33 @@ See [Secure Sensu][16] for information about cluster security.
 
 ## Use an external etcd cluster
 
-To use Sensu with an external etcd cluster, you must have etcd 3.3.2 or newer.
-To stand up an external etcd cluster, follow etcd's [clustering guide][2] using the same store configuration.
+You must have etcd 3.3.2 or newer to use Sensu with an external etcd cluster.
+Follow etcd's [clustering guide][2] using the same store configuration to stand up an external etcd cluster.
 
-In this example, you will enable client-to-server and peer communication authentication [using self-signed TLS certificates][13].
-To start etcd for `backend-1` based on the [three-node configuration example][19]:
+To initialize a backend that uses etcd authentication, configure read and write access to the `/sensu.io/` key space for your users:
+{{< highlight shell >}}
+/opt/etcd/etcdctl role grant-permission sensu_readwrite readwrite --from-key '/sensu.io/'
```

Comment:
```suggestion
# Add the sensu user, you'll be prompted for the password interactively
etcdctl user add sensu
# Create a role
etcdctl role add sensu_readwrite
# Give that role permission to read and write under the /sensu.io/ keyspace
etcdctl role grant-permission sensu_readwrite readwrite --from-key '/sensu.io/'
# Grant the sensu user the role
etcdctl user grant-role sensu sensu_readwrite
```

I found additional steps necessary here.

Reply:
This will need .initialized still too.  I just built a new cluster and it failed to init without having access to that key.

Reply:
@tarcinil do you mean we need to incorporate http://localhost:1313/sensu-go/5.20/installation/install-sensu/#3-initialize by reference? Or do you mean something else?